apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -399,7 +400,7 @@ export class CaseService {
         }),
         this.formatMessage(m.signedRuling.courtBody, {
           courtCaseNumber: theCase.courtCaseNumber,
-          linkStart: `<a href="${this.config.deepLinks.completedCaseOverviewUrl}${theCase.id}">`,
+          linkStart: `<a href="${this.config.clientUrl}${SIGNED_VERDICT_OVERVIEW_ROUTE}/${theCase.id}">`,
```

Comment:
Note that in indictments, we don't use the signed verdict overview screen. We use the Overview screen for completed cases. Does that need to be reflected here?